bin/extract-translations.sh

```diff
@@ -1,2 +1,3 @@
 #!/bin/bash
-php72 bin/console translation:extract --config=default --env=dev
+php bin/console translation:extract --env=dev nl_NL --format=xliff --force
+php bin/console translation:extract --env=dev en_GB --format=xliff --force
```

Comment:
Can the `--force` be removed here?

Reply:
` [ERROR] You must choose one of --force or --dump-messages`